test/parallel/test-child-process-fork-net.js

```diff
@@ -146,9 +149,14 @@ if (process.argv[2] === 'child') {
   server.on('close', mustCall(function() {
     closeEmitted = true;
 
-    child1.kill();
-    child2.kill();
-    child3.kill();
+    // Clean up child processes.
+    try {
+      child1.kill();
+      child2.kill();
+      child3.kill();
+    } catch {
+      debug('child process already terminated');
+    }
```

Comment:
Don't we need to split into 3 different try/catch blocks?

Reply:
I don't think so, they are only there for cleanups, we don't care about individual failures as far as I can tell.

Reply:
Good point, updated, PTAL.